codegen/projections/high_score_service/lib/high_score_service/client.rb

```diff
@@ -386,5 +386,23 @@ def output_stream(options = {}, &block)
 
       StringIO.new
     end
+
+    def transform_stubs(operation_name, stubs)
```

Comment:
I'm not a fan of this... should the class just be passed in to stubbing middleware somehow?